lib/internal/test_runner/test.js

```diff
@@ -998,7 +1000,21 @@ class Test extends AsyncResource {
       process.exit();
     }
   }
-
+  getFiles(files) {
+    if (!files) {
+      files = [];
+    }
+    if (this instanceof FileTest) {
+      const filePath = resolve(this.config.cwd, this.name);
```

Comment:
Couldn't we *not* resolve it, and use the `relativePath` in the coverage class? If not, ignore this comment.

Reply:
`getFiles()` is only called to send the files to the coverage report.

I think that the only `cwd` that applies is the root one and it is inherited by all the subtests, so is impossible to have 2 `FileTest` with the same name, but having each `FileTest` its own `config.cwd` parameter, just by reading this section of code it seems that somehow 2 `FileTest` may have different `cwd` for whatever reason, so if it returns just the files as an array of strings, I think that is better to return absolute paths, just in case the test name is not in the same `cwd` as the root.

If you ensure me that the `cwd` of the subtests are going to be the same as the root one, I can move the resolve outside of the `getFiles()` method, just before sending it to the coverage summary. In the end it will be the same as I'm only calling this `getFiles()` method there, but I guess that is more correct for this function to return whatever it says that is going to return, which are just the file names, and resolve them outside if needed.